gdx/src/com/badlogic/gdx/graphics/Texture.java

```diff
@@ -124,6 +124,14 @@ public Texture (TextureData data) {
 		load(data);
 		if (data.isManaged()) addManagedTexture(Gdx.app, this);
 	}
+
+	public Texture (TextureData data, int glHandle) {
+		super(GL20.GL_TEXTURE_2D, glHandle);
+		load(data);
+		if (data.isManaged()) {
+			addManagedTexture(Gdx.app, this);

```

Comment:
Adding a manged texture will create a new handle on context loss, which is inconsistent if you let the user specify its handle.